.github/ISSUE_TEMPLATE/new_component.md

```diff
@@ -0,0 +1,63 @@
+---
+name: New Component
+about: Interested in contributing a new component to Fluent UI React-Native? This template includes necessary information to get started, and steps to completion
+---
+
+<!-- Use this template for new components or new component variants -->
+
+## Component details
+
+### Description
+
+<!-- fill this out -->
+
+### Which product teams/scenarios need this control
+
+<!-- The more teams/scenarios that would use this control the better chance it will get prioritized -->
+
+### Design assets
+
+<!-- Please provide links to redlines or screenshots of intended component design -->
+<!-- Are you internal to Microsoft and your design isn't yet public? Please reach out to us internally via the Fluent UI Apple Teams channel or log a request at https://aka.ms/fluentui-native. -->
+
+### Component ownership
+
+<!-- Are there one or more people who can help maintain this component over time? Who will address bugs? -->
+
+## Component composition
+
+### Imports/Dependencies
+
+<!-- What other dependencies will your component be using -->
```

Comment:
what does "other" mean? I wonder if we want them to list all dependencies and not just new ones since it's likely we'll end up in many hybrid states where we're trying to get rid of dependencies and don't want new PRs to add dependencies on things we're trying to kill